src/app/cases/page.tsx

```diff
@@ -1,82 +1,51 @@
-import COLORS from '@/styles/colors';
-import { CenteredH3, H1, H2 } from '@/styles/text';
-import { CaseListing } from '@/types/schema';
-import { useAuth } from '@/utils/AuthProvider';
-import { parseAgency } from '@/utils/helpers';
-import { useProfile } from '@/utils/ProfileProvider';
-import {
-  AuthButtons,
-  Body,
-  CardColumn,
-  CaseDetailsContainer,
-  FiltersContainer,
-  Header,
-  ListingCount,
-  NoCasesContainer,
-  PageContainer,
-  ResetFilters,
-} from './styles';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { getAllCases } from '@/api/supabase/queries/cases';
+import ListingPage from '@/components/ListingPage';
+import { CaseListing, Listing } from '@/types/schema';
+import { boolToInt, nullOrUndefined } from '@/utils/helpers';
 
-type FilterType = {
-  remote: Set<string>;
-  role: Set<string>;
-  agency: Set<string>;
-  languages: Set<string>;
-  countries: Set<string>;
-};
-
-const defaultFilterValues = {
-  remote: 'Remote/In Person',
-  role: 'Roles needed',
-  languages: 'Languages',
-  agency: 'Adjudicating Agency',
-  countries: 'Country of Origin',
-};
+const remoteOptions = new Set(['Remote', 'In Person']);
 
 export default function Page() {
-  const profile = useProfile();
-  const auth = useAuth();
-  const [selectedCard, setSelecatedCard] = useState<UUID | null>(null);
   const [caseData, setCaseData] = useState<CaseListing[]>([]);
-  const [caseInfo, setCaseInfo] = useState<CaseListing>();
-  const [caseFilters, setCaseFilters] = useState<FilterType>({
-    remote: new Set(),
-    role: new Set(),
-    agency: new Set(),
-    languages: new Set(),
-    countries: new Set(),
-  });
+  const [remoteFilters, setRemoteFilters] = useState(new Set<string>());
+  const [agencyFilters, setAgencyFilters] = useState(new Set<string>());
+  const [languagesFilters, setLanguagesFilters] = useState(new Set<string>());
+  const [countriesFilters, setCountriesFilters] = useState(new Set<string>());
+  const [selectedListing, setSelectedListing] = useState<Listing | null>(null);
+
+  // load cases on first render
+  useEffect(() => {
+    (async () => {
+      try {
+        const cases = await getAllCases();
+        setCaseData(cases);
+        if (cases.length > 0) setSelectedListing(cases[0]);
+      } catch (error) {
+        console.error(error instanceof Error ? error.message : String(error));
+      }
+    })();
+  }, []);
 
-  const remoteOptions = useMemo(() => new Set(['Remote', 'In Person']), []);
-  const roleOptions = useMemo(() => new Set(['Attorney', 'Interpreter']), []);
   const agencyOptions: Map<string, string> = useMemo(
     () =>
       new Map(
         caseData
           .filter(c => c.adjudicating_agency)
-          .map(c => [
-            c.adjudicating_agency ?? '',
-            c.adjudicating_agency ? parseAgency(c.adjudicating_agency) : '',
-          ]),
+          .map(c => [c.adjudicating_agency ?? '', c.adjudicating_agency ?? '']),
```

Comment:
why do we need an array with the adjudicating_agency in 2 entries?

Reply:
tru - will make set